.php_cs.dist

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
Do this file has to be in the root? No other options like `.github` or `.travis` folders?

Then please do changes similar to https://github.com/magento/magento2/blame/develop/.htaccess#L225 in both `.htaccess` and `.htaccess.sample`.